test/Sanitizers/tsan/racy_async_let_fibonacci.swift

```diff
@@ -17,6 +17,7 @@
 // rdar://86825277
 // UNSUPPORTED: CPU=arm64e
```

Comment:
For me (also macOS arm64 with Ninja), this test also fails from time to time, so I also think this one is flaky. 

I'm by no means an expert myself, but I think it should not be disabled unconditionally, but only on arm64. To do that, you'd need to adjust this condition here. I'd say:
```suggestion
// UNSUPPORTED: CPU=arm64, CPU=arm64e
```